### DISCUSSION TITLE
v1.10.2

### DISCUSSION THREAD

**Discussion by bowring** [CIRDLES/Squid#717] [Announcements]
## What's Changed
* Updated gradle by @bowring in https://github.com/CIRDLES/Squid/pull/705
* Tweakapi by @bowring in https://github.com/CIRDLES/Squid/pull/707
* Improve WeightedMeans Handling by @bowring in https://github.com/CIRDLES/Squid/pull/710
* WM updates by @bowring in https://github.com/CIRDLES/Squid/pull/711

**Full Changelog**: https://github.com/CIRDLES/Squid/compare/v1.10.0...v1.10.2

<hr /><em>This discussion was created from the release <a href='https://github.com/CIRDLES/Squid/releases/tag/v1.10.2'>v1.10.2</a>.</em>

**Comment by NicoleRayner** [comment]
Love the drag/drop to create a new .squid file from an xlm or open an existing .squid.  Just drop it onto either the blue splash screen or the Project - Manage Project pane!
